<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>831</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Selecione o que você gostaria de editar:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editarButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>500</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voltarButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="excluirRB">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>911</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Excluir Medição</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>891</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QRadioButton" name="nomeRB">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Novo Nome do Projeto        </string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">ButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nomeText">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>921</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="alturaRB">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nova Altura                            </string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">ButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="alturaText">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>921</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="larguraRB">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nova Largura                        </string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">ButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="larguraText">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>921</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QRadioButton" name="comprimentoRB">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Novo Comprimento              </string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">ButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="comprimentoText">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>921</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="dataRB">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nova Data</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">ButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDateEdit" name="data">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>excluirRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>nomeRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excluirRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>comprimentoRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nomeRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>nomeText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excluirRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>larguraRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excluirRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>alturaRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>larguraRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>larguraText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excluirRB</sender>
   <signal>clicked(bool)</signal>
   <receiver>dataRB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comprimentoRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>comprimentoText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>data</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alturaRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>alturaText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="ButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
